### ISSUE TITLE
Security vulnerability in old version of lodash

### ISSUE THREAD

**Issue by jmattstevens** [Redocly/redoc#2528]
**Describe the bug**
The redoc/benchmark/index.html file references an obsolete version of lodash (4.17.4) with a known vulnerability that was fixed in later versions. The latest version is 4.17.21, which seems to have fixed the problem. Could the vulnerable version be replaced with the fixed version? Is the benchmark folder necessary to run redoc?

**Expected behavior**
I expected redoc to pass muster with the security team at my company, but it was rejected because of the known vulnerability. See attached file.
[Redoc ML-vulnerability-report.xlsx](https://github.com/Redocly/redoc/files/15099577/Redoc.ML-vulnerability-report.xlsx)

**Minimal reproducible OpenAPI snippet(if possible)**

**Screenshots**

**Additional context**

**Comment by jmattstevens** [comment]
So if I use NPM to install redoc, the benchmark/index.html file is not included?On Apr 25, 2024 8:19 AM, Alex Varchuk ***@***.***> wrote:
we do not include redoc/benchmark/index.html to lib code. Later we'll fix that

**Comment by AlexVarchuk** [comment]
yes, you can check it in file list. For example on [npm/redoc](https://www.npmjs.com/package/redoc?activeTab=code)
